packages/fxa-settings/src/components/SecurityPanel/index.tsx

```diff
@@ -0,0 +1,63 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { UnitRow } from '../UnitRow';
+
+type SecurityPanelProps = {
+  twoFactorAuth?: object;
+  accountRecoveryKey?: object;
+  className?: string;
+};
+
+export const SecurityPanel = ({
+  twoFactorAuth,
+  accountRecoveryKey,
+  className,
+}: SecurityPanelProps) => {
+  const getValue = (settingOption?: object) =>
```

Comment:
`getValue` and `getClassName` gets us a head start in #4910 and #4913. You could remove these and hardcode the values to close this issue, otherwise we're probably going to want to require props like:

```
type SecurityPanelProps = {
  twoFactorAuthEnabled: boolean;
  accountRecoveryKeyEnabled: boolean;
  className?: string;
};
```

Then on L128 in `Settings/index.tsx`, you can pass in the real data there from `accountData` to get us further ahead on those two issues, or just pass in fake data (probably `false` since the ticket asks for "Not set" to be hardcoded).